scripts/mkcloud

```diff
@@ -582,7 +587,6 @@ function createcloudsnapshot()
         create_snapshot_volume "${cloudvg}/${d}"
     done
     restartcloud
-    wait_for_crowbar_ssh
```

Comment:
I will not remove this, for me makes sense to have two synchronization points, because there are other variables that can fail, not only ntpd.